apps/labs/posts/lets-make-pygments-accessible.md

```diff
@@ -0,0 +1,70 @@
+---
+title: '"Let's make Pygments accessible!"'
```

Comment:
```suggestion
title: "Let's make Pygments accessible!"
```

Reply:
I think the double quotes might be the issue here.

Reply:
It was with single quotes before so will need inspecting closer

Reply:
Sorry, poorly phrased! I meant double as in double within single.

Reply:
The author slug is correct. I checked that before commenting. This is what made me investigate the row above to begin with.

Reply:
Yup we have tried few combos of the quotes. 

@isabela-pf and @steff456  since it has been too much faff debugging this what about changing the title to "Making pygments accessible" 🙏🏽 ? At least to avoid delaying this once more